windows/bleachbit.nsi

```diff
@@ -357,9 +357,16 @@ SectionGroup /e Shortcuts
     SectionEnd
 
     Section "Desktop" SectionDesktop
+        ${GetParameters} $R0
+        ${GetOptions} $R0 "/S /NoDesktopShortcut" $R0
```

Comment:
@rados 
Is there an easy way to make this (1) not case sensitive and (2) flexible to allow other orders such as `/S /AllUsers /NoDesktopShortcut`?